site/content/en/docs/tutorials/setup_minikube_in_china.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "minikube start"
+linkTitle: "Get Started!"
+weight: 1
+---
+
+This Tutorial explains how to setup minikube in China.
+
+<h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">1</strong></span>Installation</h2>
+
+### Binary download
+
+For more detail please refer to [minikube installation guide]({{<ref "/docs/start">}})
+
+Open a terminal. If on Windows, open a terminal with administrator access. Run:
+
+```shell
+minikube start --image-repository=registry.cn-hangzhou.aliyuncs.com/google_containers
+```
+If minikube fails to start, see the [drivers page]({{<ref "/docs/drivers">}}) for help setting up a compatible container or virtual-machine manager.
+
+For users who use the Hyper-V environment, you should first open the Hyper-V manager to create an external virtual switch. After that, we can use the following commands to create a Hyper-V-based Kubernetes test environment:
```

Comment:
Can you point users to our HyperV docs here 

https://minikube.sigs.k8s.io/docs/drivers/hyperv/

Reply:
updated.